addons/hr_evaluation/__openerp__.py

```diff
@@ -20,47 +20,45 @@
+    'summary': 'Periodical Evaluations',
+    'images': ['images/hr_evaluation_analysis.jpeg',
+               'images/hr_evaluation.jpeg',
+               'images/hr_interview_requests.jpeg'],
     'depends': ['hr', 'calendar', 'survey'],
     'description': """
-Periodical Employees evaluation and appraisals
+Periodical Employees evaluation
 ==============================================
 
 By using this application you can maintain the motivational process by doing periodical evaluations of your employees' performance. The regular assessment of human resources can benefit your people as well your organization.
 
-An evaluation plan can be assigned to each employee. These plans define the frequency and the way you manage your periodic personal evaluations. You will be able to define steps and attach interview forms to each step.
-
-Manages several types of evaluations: bottom-up, top-down, self-evaluations and the final evaluation by the manager.
+An evaluation plan can be assigned to each employee. These plans define the frequency and the way you manage your periodic personal evaluations.
 
 Key Features
 ------------
 * Ability to create employees evaluations.
-* An evaluation can be created by an employee for subordinates, juniors as well as his manager.
+* An evaluation can be created by an employee's manager or automatically based on schedule which defined in employee form.
 * The evaluation is done according to a plan in which various surveys can be created. Each survey can be answered by a particular level in the employees hierarchy. The final review and evaluation is done by the manager.
-* Every evaluation filled by employees can be viewed in a PDF form.
-* Interview Requests are generated automatically by OpenERP according to employees evaluation plans. Each user receives automatic emails and requests to perform a periodical evaluation of their colleagues.
+* Manager, colleague, collaborator, and self receives automatic email to perform a periodical evaluation when appraisal is started.
+* Every evaluation/Appraisal Form filled by employees, colleague, collaborator, can be viewed in a PDF form.
+* Meeting Requests are created manually according to employees evaluation plans.
 """,
     "data": [
         'security/ir.model.access.csv',
         'security/hr_evaluation_security.xml',
-        'hr_evaluation_view.xml',
+        'views/hr_evaluation_view.xml',
         'report/hr_evaluation_report_view.xml',
-        'survey_data_appraisal.xml',
-        'hr_evaluation_data.xml',
-        'hr_evaluation_installer.xml',
+        'data/survey_data_appraisal.xml',
+        'data/hr_evaluation_data.xml',
+        'views/hr_evaluation.xml',
     ],
-    "demo": ["hr_evaluation_demo.xml"],
-    # 'test': [
-    #     'test/test_hr_evaluation.yml',
-    #     'test/hr_evalution_demo.yml',
-    # ],
-    'auto_install': False,
+    "demo": ["data/hr_evaluation_demo.xml"],
+    'test': ['test/test_hr_evaluation.yml'],
```

Comment:
Convert the yml tests into python tests (as they are only executing python anyway)